src/t8_cmesh/t8_cmesh.c

```diff
@@ -107,8 +107,7 @@ t8_cmesh_is_committed (const t8_cmesh_t cmesh)
 
 #ifdef T8_ENABLE_DEBUG
     /* TODO: check more conditions that must always hold after commit */
-    if ((!t8_cmesh_trees_is_face_consistent (cmesh, cmesh->trees)) || (!t8_cmesh_no_negative_volume (cmesh))
-        || (!t8_cmesh_check_trees_per_eclass (cmesh))) {
+    if ((!t8_cmesh_trees_is_face_consistent (cmesh, cmesh->trees)) || (!t8_cmesh_check_trees_per_eclass (cmesh))) {
```

Comment:
I think we should move the face-consistency and the the trees per eclass should be moved to the is_valid check, too.

Reply:
The plan was to just separate the geometry checks